kong/plugins/zipkin/tracing_headers.lua

```diff
@@ -314,6 +315,41 @@ local function parse_jaeger_trace_context_headers(jaeger_header)
 end
 
 
+local function parse_datadog_headers(headers)
+  -- allow testing to spy on this.
```

Comment:
I think this comment is incorrect. The spy will work even if `warn` is not localized. It's just a little more efficient to localize it than to not localize it.
```suggestion
```